install.sh

```diff
@@ -1,15 +1,13 @@
+#!/bin/bash
```

Comment:
Change this to `#!/usr/bin/env bash` (also for the others, but keep `sh` as `sh`)